complete/src/main/java/com/example/restservice/service/UserService.java

```diff
@@ -21,12 +21,12 @@
 
   public UserStatusResponse getStatus(String tokenId) {
     return new UserStatusResponse(
+        getAheadCount(tokenId).orElseThrow(SQInternalServerException::new),
         tokenId,
         userRepository
-            .findById(tokenId)
-            .orElseThrow(SQInvalidRequestException::userNotFoundException)
-            .getStatus(),
-        getAheadCount(tokenId).orElseThrow(SQInternalServerException::new));
+                .findById(tokenId)
+                .orElseThrow(SQInvalidRequestException::userNotFoundException)
+                .getStatus());
   }
```

Comment:
Here and above, I feel that the previous ordering makes more sense. Can you rearrange the member definitions in the classes to restore the constructor ordering?

Reply:
restored ordering on latest commit